packages/internal/src/debug.js

```diff
@@ -1,41 +1,108 @@
 // @jessie-check
 
+/**
+ * @import {OptPrefixPath} from './types-internal.js';
+ */
+
+// TODO This is top level static mutable state. Why did it pass at-jessie-check?
```

Comment:
```suggestion
// XXX This is top level static mutable state. Why did it pass at-jessie-check?
```

or phrase it as something to do, preferably with an issue number / link

see [kinds of comments](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments)

Reply:
Obsolete. Now documented at #11844 . Resolving.